inhouse_bot/dataclasses/GameInfo.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
```

Comment:
To be consistent with GameInfo, you should add the Teams dataclass (see game.py) as well

Reply:
what do you mean here?

Reply:
[https://github.com/adamschachne/inhouse_bot/blob/72-refactoring-matchmaking/inhouse_bot/database_orm/tables/game.py#L64](game.py) Extract this dataclass into a new file

Reply:
oh ok